core/chains/evm/config/toml/config.go

```diff
@@ -451,6 +452,20 @@ func (c *Chain) ValidateConfig() (err error) {
 					err = multierr.Append(err, commonconfig.ErrInvalid{Name: "GasEstimator.BumpThreshold", Value: 0, Msg: fmt.Sprintf("cannot be 0 if Transactions.AutoPurge.MinAttempts is set for %s", chainType)})
 				}
 			}
+		case chaintype.ChainDualBroadcast:
```

Comment:
Could you please update the PR description? Right now it just says "improvements", whereas, at least it takes care of the ChainDualBroadcast chain type. 

Please, also specify what is the old vs new behavior after this PR is merged.